File: 프로그래머스/lv0/120818. 옷가게 할인 받기/옷가게 할인 받기.java
import java.math.*;
class Solution {
    public int solution(int price) {
        int answer = 0;
        if(price >=500000){
            answer=(int)Math.floor(price*0.8);
            
        }else if(price >=300000){
            answer=(int)Math.floor(price*0.9);
        }else if(price>=100000){
            answer=(int)Math.floor(price*0.95);
        }else{
            answer=price;
        }
       // System.out.println((int)Math.floor(price*0.95));
        return answer;
    }
}

File: 프로그래머스/0/120818. 옷가게 할인 받기/옷가게 할인 받기.py
import math
def solution(price):
    answer = 0
    if (price < 100000):
        answer=price
    else:
        if(price>=500000):
             answer=math.floor(price*0.8)
        elif (price>=300000):
             answer=math.floor(price*0.9)
        elif(price>=100000) :
             answer=math.floor(price*0.95)   
    return answer